veeam/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
veeam/client.py,sha256=jXC5F4GjUOAgUlvmAGw3fDxm82QGJNK5YNwU29uJIMQ,9099
veeam/errors.py,sha256=Lie30vdP3_7iYrO6yOYL0qsIeJIvRTJCyqfRR7rEa2g,706
veeam-0.1.5.dist-info/METADATA,sha256=77EM9jgOZcKdFk29y8RBEHz0Hf6yCuzPNRhRbAKakHw,1880
veeam-0.1.5.dist-info/WHEEL,sha256=S8S5VL-stOTSZDYxHyf0KP7eds0J72qrK0Evu3TfyAY,92
veeam-0.1.5.dist-info/top_level.txt,sha256=DY6TIwTOXYMMO4tmKLVpJIftzgD5oExnlFA_voM569s,6
veeam-0.1.5.dist-info/RECORD,,
